README.md

```diff
@@ -1,13 +1,13 @@
-<a href="http://promisesaplus.com/">
-    <img src="http://promisesaplus.com/assets/logo-small.png" alt="Promises/A+ logo"
-         title="Promises/A+ 1.1 compliant" align="right" />
-</a>
+# ⚠️Please use native promises instead if at all possible. ⚠️ 
```

Comment:
Love to see this reflected on the [website](http://bluebirdjs.com/docs/getting-started.html) as well.